addons/im_livechat/static/src/components/thread_icon/thread_icon_tests.js

```diff
@@ -5,85 +5,50 @@ const components = {
     ThreadIcon: require('mail/static/src/components/thread_icon/thread_icon.js'),
 };
 const {
-    afterEach: utilsAfterEach,
+    afterEach,
     afterNextRender,
-    beforeEach: utilsBeforeEach,
-    start: utilsStart,
+    beforeEach,
+    createRootComponent,
+    start,
 } = require('mail/static/src/utils/test_utils.js');
 
 QUnit.module('im_livechat', {}, function () {
 QUnit.module('components', {}, function () {
 QUnit.module('thread_icon', {}, function () {
 QUnit.module('thread_icon_tests.js', {
     beforeEach() {
-        utilsBeforeEach(this);
+        beforeEach(this);
 
         this.createThreadIcon = async thread => {
-            const ThreadIconComponent = components.ThreadIcon;
-            ThreadIconComponent.env = this.env;
-            this.component = new ThreadIconComponent(null, { threadLocalId: thread.localId });
-            await this.component.mount(this.widget.el);
+            const props = { threadLocalId: thread.localId };
+            await createRootComponent(this, components.ThreadIcon, props, this.widget.el);
```

Comment:
The only reasoning I applied here was to use variables when the line would be longer than 100 char without it.

But I think the suggestion is fine, this function can be changed to take named arguments. Only `props` and `target`? All of them?

Reply:
I'm good with just `props` and `target`. `this` should definitely stay positional 1st argument, and the Component can stay as 2nd positional since this is always required and it has direct meaning with the name of the helper function.

Reply:
Done